modoptions.lua

```diff
@@ -302,7 +314,6 @@ local options = {
 		type	= "bool",
 		section	= "options_main",
 		def		= false,
-		column	= 1,
```

Comment:
What happened with these column changes? If default is 1 we still want to keep no changes, if a change was needed can you clarify?

Reply:
Haha you caught me. I was in the process of fixing this/making the UI less garbage.